src/net/p2p.rs

```diff
@@ -912,7 +914,8 @@ impl PeerNetwork {
             outbound_neighbors.len(),
             &outbound_dist
         );
-
+        update_outbound_neighbors(outbound_neighbors.len() as i64); //promserver
+        update_inbound_neighbors(inbound_neighbors.len() as i64); //promserver
```

Comment:
This probably isn't what you want.  The `outbound_neighbors` and `inbound_neighbors` here subsets of the total inbound/outbound set sizes, since nodes don't broadcast messages to all peers.  I think you'll want to move these counters to the top-level P2P runloop, right after the call to `prune_connections()`.  You'd use the `count_outbound_conversations()` method to count the number of outbound neighbors, and then take `self.peers.len() - self.count_outbound_conversations(..)` to get the number of inbound neighbors.

Reply:
Moved: https://github.com/wileyj/stacks-blockchain/commit/192b66fe0bd669fb7bf3177034256d5383c4bdaa

Also moved `update_burnchain_height`, since it wasn't  reliably being updated on restart. Seems to work better now and burn_height is being reported reliably after a restart.